.gitignore

```diff
@@ -1,5 +1,6 @@
 # Ignore Gradle project-specific cache directory
 .gradle
+.idea/
```

Comment:
Directories can be ignored without the '/'

Reply:
Fixed this issue!